Make tracked time representation display as hours

vsysoev opened the pull request:
Since #25213 is not moving forward, I propose another minimal approach to change tracked time representation.

Tracked time in the issue represented not in the same manner as estimated.

Because of estimated time represented in hours it might be convenient to have tracked time represented in the same way to be compared and managed.

Option TRACKED_TIMESTAMP_TENSE added to [ui] section.

[ui]
TRACKED_TIMESTAMP_TENSE='mixed|hours'

wxiaoguang commented:
Maybe it should be a per-user setting? It only affects the display, so it's user's preference?

vsysoev commented:
> Maybe it should be a per-user setting? It only affects the display, so it's user's preference?

@wxiaoguang, thank you for your review. My points are:

- Main: Estimation/tracked (time or points) values should be in the same view mode to be comparable. They are used to define team/contributor performance and planning poker. 
- Existing approach to show time in gitea. There is UI.PREFERED_TIMESTAMP_TENSE option exists. My PR based on the same approach.
- Usage model in my company is standalone installations for the dev teams. It is not cloud installation for the unpredictable amount contributors.

wxiaoguang commented:
Thank you for the explanation. TBH if there is a chance to avoid a new global config option, I think it's better to do that, meanwhile I won't block the current approach.

But I can see there are other related PRs:

*  WIP: Add Sec2TrackedTime time format template helper #25213 (as you mentioned: it is not moving forward)
*  WIP: Replaced Sec2Time with Sec2Hours for better timesheets reporting. #28312 (it is yours, what's the relation of them?)

And @6543 to take a look at this PR and #25213

vsysoev commented:
> Thank you for the explanation. TBH if there is a chance to avoid a new global config option, I think it's better to do that, meanwhile I won't block the current approach.
> 
> But I can see there are other related PRs:
> 
> * WIP: Add Sec2TrackedTime time format template helper [WIP: Add Sec2TrackedTime time format template helper #25213](https://github.com/go-gitea/gitea/pull/25213) (as you mentioned: it is not moving forward)
> * WIP: Replaced Sec2Time with Sec2Hours for better timesheets reporting. [WIP: Replaced Sec2Time with Sec2Hours for better timesheets reporting. #28312](https://github.com/go-gitea/gitea/pull/28312) (it is yours, what's the relation of them?)
> 
> And @6543 to take a look at this PR and #25213

Yes, I took a look on #25213 and doing several comments in this PR. But #25213 isn't moving forward for more than a year. 
#28312 is my "old patch" which I use for my installations release 1.20+. In 1.23.1 I found than estimation field have been added. So I see that #28312 is outdated and I close it. I reimplement patch for my needs and propose it to upstream.

wxiaoguang commented:
Maybe it is a good chance to complete remove the legacy `SecToTime`, it doesn't make sense to convert 24hours to one day.

vsysoev commented:
> Maybe it is a good chance to complete remove the legacy `SecToTime`, it doesn't make sense to convert 24hours to one day. Then no need to introduce the new option.

If it will be suitable I remove SecToTime and replaced it with SecToHours.

wxiaoguang commented:
We also need to fix frontend: f33378694d17d0aacc74fa34e7e606df5aac79a3
